activemodel/lib/active_model/translation.rb

```diff
@@ -62,11 +64,13 @@ def human_attribute_name(attribute, options = {})
 
       defaults << :"attributes.#{attribute}"
       defaults << options[:default] if options[:default]
-      defaults << MISSING_TRANSLATION
+      defaults << MISSING_TRANSLATION unless Translation.raise_on_missing_translations
```

Comment:
Hum, I'm sorry I was wrong. Here you need to consider both `option` and `Translation.raise_on_missing_translations`

```suggestion
      defaults << MISSING_TRANSLATION unless options.fetch(:raise, Translation.raise_on_missing_translations)
```

Reply:
Thank you!